gems/quilt_rails/test/quilt_rails/react_renderable_test.rb

```diff
@@ -59,5 +66,8 @@ def reverse_proxy(url, headers: {})
     def form_authenticity_token
       'foo'
     end
+
+    # Stubbing request that exist in a controller
+    attr_reader :request
```

Comment:
```suggestion
    def request
      @request ||= ActionDispatch::TestRequest.create.tap do |request|
        request.request_id = SecureRandom.uuid
      end
    end
```
And then `request.request_id` in tests if you want to shorthand what the setup block does.